components/CommentCard.jsx

```diff
@@ -1,27 +1,66 @@
-import { useState } from "react"
+import { useState, useEffect } from "react";
+import { fetchCommentsByArticleId } from "../utils/api";
 
-export default function CommentCard({ comment }) {
-    const [votes, setVotes ] = useState(comment.votes)
+export default function CommentCard({ article_id }) {
+  const [comments, setComments] = useState([])
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
 
-    const handleUpVote = () => {
-        setVotes(votes + 1)
-    }
+  useEffect(() => {
+    setIsLoading(true);
+    setIsError(false);
+    fetchCommentsByArticleId(article_id)
+      .then((commentData) => {
+        setComments(commentData);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setIsError(true);
+        setIsLoading(false);
+      });
+  }, [article_id]);
 
-    const handleDownVote = () => {
-        setVotes(votes - 1)
-    }
+  const handleVote = (commentId, increment) => {
+    setComments((prevComments) =>
+      prevComments.map((comment) =>
+        comment.comment_id === commentId
+          ? { ...comment, votes: comment.votes + increment }
+          : comment
+      )
+    );
+  };
+
+  if (isError) {
+    return <p>ERROR 404 - Content not found</p>;
+  }
+  if (isLoading) {
+    return <div>LOADING!</div>;
+  }
 
   return (
-    <li key={comment.comment_id}>
-      <p>
-        <strong>{comment.author}</strong> on{" "}
-        {new Date(comment.created_at).toLocaleDateString()}
-      </p>
-      <p>{comment.body}</p>
-      <p>Votes: {votes}</p>
-      <button onClick={handleUpVote}>+1 vote</button>
-      <button onClick={handleDownVote}>-1 vote</button>
-    </li>
-  );
+    <ul>
+        {comments.length > 0 ? (
+            comments.map((comment) => (
+                <li key={comment.comment_id}>
+                    <p>
+                        <strong>{comment.author}</strong> on{" "}
```

Comment:
You need to be sure you are using strong for the right reasons, see the docs. If it just to make the text bold use `<b>` or CSS